tests/disposableredis/__init__.py

```diff
@@ -38,14 +38,17 @@ def retry_with_rdb_reload(self):
 
 class DisposableRedis(object):
 
-    def __init__(self, port=None, path='redis-server', **extra_args):
+    def __init__(self, port=None, path=None, **extra_args):
         """
         :param port: port number to start the redis server on.
             Specify none to automatically generate
         :type port: int|None
         :param extra_args: any extra arguments kwargs will
             be passed to redis server as --key val
         """
+        if path is None:
+            path = REDIS_SERVER
```

Comment:
log incase path is None and REDIS_SERVER isn't defined.

Reply:
REDIS_SERVER is always defined: os.environ.get('REDIS_SERVER', 'redis-server') - 2nd arg is default value. However, we can employ protective programming in this case.